### ISSUE TITLE
Notice 00100 in every Guild Member

### ISSUE THREAD

**Issue by xJimmyDeluxex** [stpain/guildbook#102]
We are using the Guild book in our guild and it works fine but there is an 00100 notice in ever Guild members tab eventually placed by this Addon. If ever this notice is used by Guildbook or accidentally written. Is it possible to turn off this feature? Thanks a lot.

**Comment by Belrand** [comment]
Hey there,
Same issue had been reported two months ago, unfortunately it's not caused by Guildbook but rather a DKP addon.
From Discord:
![](https://media.discordapp.net/attachments/855511849222275075/940272896624967701/unknown.png)

Guildbook doesn't write anything in the guild notes, it only "reads" them

**Comment by stpain** [comment]
Yeah it's strange how guildbook gets caught up in this but as Belrand said it doesn't write or change any data anywhere only scans and reads info

**Comment by xJimmyDeluxex** [comment]
Thanks for the quick answer. 
It was just a thought that Guild book caused this bug because it appeared since some of our members used Guildbook. As you said Guildbook doesn`t write any notes so we will try track to addon who caused this problem.
